javascript/modules/client/src/internal/interfaces/plugins.ts

```diff
@@ -175,10 +194,58 @@ export type Erc20Proposal = Proposal & {
   open: boolean;
   executed: boolean;
 };
-
 export type Erc20Token = {
   address: string;
   name: string;
   symbol: string;
   decimals: number;
 };
+export interface IERC20ProposalQueryParams extends IPagination {
```

Comment:
If this is generic and applicable to other plugins, I would factor it out and call it `IProposalQueryParams`

Reply:
Sure, this can be shared between `ERC20Voting` and `AllowListVoting`